next/utils/chunkifyArray.ts

```diff
@@ -0,0 +1,7 @@
+export function chunkifyArray<T>(array: T[], size: number): T[][] {
```

Comment:
Please use what we already have installed.
https://lodash.com/docs/#chunk

Reply:
perfect!